### ISSUE TITLE
Add options to set viewport width/height? Ignore viewport meta?

### ISSUE THREAD

**Issue by PinkiNice** [pichillilorenzo/flutter_inappwebview#682]
## Description

Couldn't find a setting to set viewport (screen) width/height in `HeadlessInAppWebView`. Is it possible?  I can force specific size for `InAppWebView` using `SizedBox`, but that is not an option when using `HeadlessInAppWebView`.

I tried to ignore meta[name="viewport"] tag somehow, using `loadWithOverviewMode: true` and `useWideViewPort: false`. But that didn't work as I wanted. Tried to remove meta tag with `evaluateJavascript` but evaluating js only possible after webview has been rendered, so it had no effect. Also tried to find some way to intercept received HTML and edit it before rendering, but found nothing related in documentation.
 
Basically this feature is needed to force pages to render as Desktop versions instead of Mobiles. Because `UserPreferredContentMode.DESKTOP` with desktop user agent is not always enough for that.

Any ideas? Am I missing something?

**Comment by pichillilorenzo** [comment]
Starting from version `5.3.0`, you can set the width and height of `HeadlessInAppWebView` using the `initialSize` property or the `setSize` method.

Example:
```dart
var headlessWebView = new HeadlessInAppWebView(
    initialUrlRequest: URLRequest(url: Uri.parse("https://github.com/flutter")),
    initialSize: Size(600, 800),
);
```
